packages/mobile/src/App.tsx

```diff
@@ -58,6 +59,11 @@ const Modals = () => {
 }
 
 const App = () => {
+  // Reset libs so that we get a clean app start
+  useEffectOnce(() => {
```

Comment:
wondering if we want to do `useLayoutEffect` here to make sure it happens before the first render? i think with [] deps on it, it should only once yeah?

Reply:
I can change it yeah, don't think it makes much of a difference though because I was seeing App render multiple times before the sagas even started running

Reply:
oh chill sg, no worries then